### PR TITLE
Assert active power, reactive power and current consistency

### PR DESCRIPTION
**Pull request by geofjamg**
Signed-off-by: Geoffroy Jamgotchian <geoffroy.jamgotchian@gmail.com>

**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
No

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*
Unit test

**What is the new behavior (if this is a feature change)?**
A unit test has been added to assert p, q and i equations consistency

**Does this PR introduce a breaking change or deprecate an API?** *If yes, check the following:*
- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: src/test/java/com/powsybl/openloadflow/ac/AcEquationsTest.java [hunk 1]
```diff
@@ -141,6 +142,28 @@ void branchTest() {
...
+        var p1Eq = new ClosedBranchSide1ActiveFlowEquationTerm(branch, bus1, bus2, variableSet, true, true);
+        p1Eq.setStateVector(sv);
+        double p1 = p1Eq.eval();
+        var q1Eq = new ClosedBranchSide1ReactiveFlowEquationTerm(branch, bus1, bus2, variableSet, true, true);
+        q1Eq.setStateVector(sv);
+        double q1 = q1Eq.eval();
+        var i1Eq = new ClosedBranchSide1CurrentMagnitudeEquationTerm(branch, bus1, bus2, variableSet, true, true);
+        i1Eq.setStateVector(sv);
+        double i1 = i1Eq.eval();
+        assertEquals(i1, Math.hypot(p1, q1) / V_1, 10e-14);
```
Comment (geofjamg) [inline src/test/java/com/powsybl/openloadflow/ac/AcEquationsTest.java:155]: @annetill is it normal to not have the sqrt(3) here ?

Comment (annetill) [reply src/test/java/com/powsybl/openloadflow/ac/AcEquationsTest.java:155]: After looking at the code, yes it is normal. The sqrt(3) is managed by function PerUnit.ib() for sensitivity and security analysis.

Comment (geofjamg) [reply src/test/java/com/powsybl/openloadflow/ac/AcEquationsTest.java:155]: It doesn't mean it is correct. As far as I understand main principle of peruniting is to perunit only 2 quantities (for us it is apparent power and voltage) and all the other quantities are calculated from the 2 chosen ones.
So for me, the values in per unit are not correct even if one in SI are correct.

Comment (geofjamg) [reply src/test/java/com/powsybl/openloadflow/ac/AcEquationsTest.java:155]: According https://en.wikipedia.org/wiki/Per-unit_system it seems correct

**Review by annetill — APPROVED** [review]

**Pull request merged**